README.md

```diff
@@ -5,7 +5,12 @@ Extra manifests for [Scoop](https://scoop.sh), the Windows command-line installe
 To make it easier to install apps from this bucket, run
     `scoop bucket add extras`
 
-## contributing
-For documentation on how to add packages to the Extras bucket see [the Wiki](https://github.com/ScoopInstaller/Scoop/wiki) from main [Scoop](https://github.com/ScoopInstaller/Scoop) project, particularly: 
+## How do I install these manifests?
+
+To add this bucket, run `scoop bucket add extras`. To install, do `scoop install <manifest>`.
+
+## Contributing
+For documentation on how to add packages to the Extras bucket see [the Wiki](https://github.com/ScoopInstaller/Scoop/wiki) from main [Scoop](https://github.com/ScoopInstaller/Scoop) project, particularly:
```

Comment:
Using the word `main` here could be slightly confusing as we have a repo/bucket named `main` as well.